assets/javascripts/kitten/components/carousel/carousel-inner.js

```diff
@@ -164,39 +145,25 @@ class CarouselInnerBase extends React.Component {
       numColumns,
       numPages,
       itemMarginBetween,
-      siblingPageVisible,
     } = this.props
 
     const rangePage = createRangeFromZeroTo(numPages)
 
     return (
-      <div
-        ref={div => {
-          this.carouselInner = div
+      <StyledCarouselInner
+        ref={node => {
```

Comment:
React propose maintenant `React.createRef()` pour mieux gérer les refs : https://reactjs.org/docs/refs-and-the-dom.html#creating-refs